packages/sitecore-pipelines/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "../../.eslintrc",
+  "rules": {
+    // TODO: fix errors
```

Comment:
Same here - wouldn't consider this a TODO

Reply:
Removed